### PR TITLE
Store all network validator shares & operators

### PR DESCRIPTION
**Pull request by olegshmuelov**
The main purpose of this PR is to store all network validators & operators when parsing ETH1 events.
After storing the validators, determine if the share belongs to operator, if so start them, otherwise ignore

### CODE REVIEW

File: cli/operator/node.go [hunk 1]
```diff
@@ -112,11 +113,15 @@ var StartNodeCmd = &cobra.Command{
 		if err := operatorStorage.SetupPrivateKey(cfg.GenerateOperatorPrivateKey, cfg.OperatorPrivateKey); err != nil {
 			Logger.Fatal("failed to setup operator private key", zap.Error(err))
 		}
-		operatorPrivKey, found, err := operatorStorage.GetPrivateKey()
+		operatorPrivateKey, found, err := operatorStorage.GetPrivateKey()
 		if err != nil || !found {
 			Logger.Fatal("failed to get operator private key", zap.Error(err))
 		}
-		cfg.P2pNetworkConfig.OperatorPrivateKey = operatorPrivKey
+		operatorPubKey, err := rsaencryption.ExtractPublicKey(operatorPrivateKey)
```
[the remark this answers is not in the document]
Comment (olegshmuelov) [reply cli/operator/node.go:120]: The operator public key is injected to validator controller in order to determine if the share belongs to operator when you start the validator. 
Extracting the operator public key each time when ValidatorAddedEvent is triggered is time consuming. This why was decided to inject the pubkey to controller.

File: eth1/client.go [hunk 2]
```diff
@@ -21,8 +21,12 @@ type Options struct {
 
 // Event represents an eth1 event log in the system
 type Event struct {
-	Log  types.Log
+	// Log is the raw event log
+	Log types.Log
+	// Data is the parsed event
 	Data interface{}
+	// IsOperatorEvent indicates whether the event belongs to operator
+	IsOperatorEvent bool
```
[the remark this answers is not in the document]
Comment (olegshmuelov) [reply eth1/client.go:29]: Asa  comment suggests, IsOperatorEvent indicates whether the event belongs to operator or not, whether its related to ValidatorAddedEvent or OperatorAddedEvent or upcoming v2 events.
We can change the name if its confusing

File: operator/storage.go [hunk 3]
```diff
@@ -57,7 +57,7 @@ func (s *storage) ListOperators(from int64, to int64) ([]registrystorage.Operato
 }
 
 func (s *storage) CleanRegistryData() error {
-	return s.cleanSyncOffset()
+	return s.db.RemoveAllByCollection(prefix)
```
Comment (amir-blox) [inline operator/storage.go:60]: The operator private key is also under this prefix, you need to clear the sync offset prefix and operators prefix

File: cli/operator/node.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (olegshmuelov) [reply cli/operator/node.go:120]: opened a task to handle the operator private/public in a proper way
https://github.com/bloxapp/ssv/issues/554

**Pull request merged**